stable/influxdb/values.yaml

```diff
@@ -0,0 +1,146 @@
+## influxdb image version
+## ref: https://hub.docker.com/r/library/influxdb/tags/
+image:
+  repo: "influxdb"
+  tag: "1.1.0-alpine"
+  pullPolicy: IfNotPresent
+
+## Specify a service type
+## NodePort is default
+## ref: http://kubernetes.io/docs/user-guide/services/
+service:
+  type: NodePort
```

Comment:
Can we make this a ClusterIP service by default? Does influx require it to be a NodePort? I'd prefer to keep services cluster internal by default so that they aren't accidentally exposed to the outside world.